src/main/scala/uk/gov/hmrc/play/http/HeaderNames.scala

```diff
@@ -43,6 +43,8 @@ object HeaderNames {
   val googleAnalyticUserId  = "ga-user-cookie-id"
   val deviceID  = "deviceID" // not a typo, should be ID
   val akamaiReputation = "Akamai-Reputation"
+  val hmrcLang  = "HMRC_LANG"
```

Comment:
NGINX blocks headers with underscores, so this will not work other than locally.

Also, please could we start adopting a standardised format for custom headers. I'd prefer if this was called `MDTP-Language` (and any future custom headers also use the `MDTP-` prefix. I'll write a page on Confluence about it for any future headers we might add.